gut_to_soil_manuscript_figures/scripts/plot_pcoa_2d.py

```diff
@@ -62,8 +63,7 @@ def _bucket_util(highlighted_buckets, md, ord_2d):
         # HE
         bucket_ids_HE_week0 = \
             md[(md['Bucket'] == bucket) &
-               (md['SampleType2'] == 'Self Sample') &
-               (md['Week'] == 0.0)].index.values
+               (md['SampleType'] == 'Human Excrement')].index.values
```

Comment:
It looks like the `(md['Week'] == 0.0)].index.values` got removed. Was this intentional?